core/services/synchronization/explorer_client.go

```diff
@@ -166,6 +163,8 @@ func (ec *explorerClient) Send(data []byte, messageTypes ...int) {
 	select {
 	case send <- data:
 		atomic.StoreUint32(&ec.dropMessageCount, 0)
+	case <-ctx.Done():
```

Comment:
I think this one isn't really needed because of the `default` but just there for symmetry with `Receive` (where it does seem actually needed) ?

Reply:
To avoid logging the buffer full message if the Send gets shut down.